tests/formattedcode/test_output_spdx.py

```diff
@@ -127,7 +127,6 @@ def check_rdf_scan(expected_file, result_file, regen=REGEN_TEST_FIXTURES):
     else:
         with io.open(expected_file, encoding='utf-8') as i:
             expected = json.load(i)
```

Comment:
The "expected" rdf files have already undergone the `load_and_clean_rdf()` process. Especially they are not in xml format anymore, so `load_and_clean_rdf(expected_file)` raises an error, as the function expects rdf-xml.
As can be seen by the green tests, the current code works fine because `expected_file` is already in json format and cleaned.

Reply:
Ah I see the issue, my bad. Thanks for the explanation.